crates/harp/src/size.rs

```diff
@@ -27,28 +27,28 @@ use crate::r_typeof;
 // `utils::object.size()` is too slow on large datasets and this code path is used trough the
 // variables pane which required more performance.
 // See for more info.
-pub fn r_size(x: SEXP) -> usize {
+pub fn r_size(x: SEXP) -> harp::Result<usize> {
     let mut seen: HashSet<SEXP> = HashSet::new();
 
     let sizeof_node: f64 = harp::parse_eval0(
         "as.vector(utils::object.size(quote(expr = )))",
         R_ENVS.global,
     )
```

Comment:
while you're here, just make this `parse_eval_global()` to match `sizeof_vector`?

Reply:
I made it `parse_eval_base()` which is more adequate here.